packages/plugins/package.json

```diff
@@ -19,7 +19,7 @@
     "copy-hardhat-config": "cp src/hardhat.config.js dist/hardhat.config.js",
     "clean": "rimraf out/ dist/ ./tsconfig.tsbuildinfo ./forge-cache",
     "test": "yarn kill-nodes && yarn test:solc && yarn test:ts",
-    "test:ts": "npx mocha --require ts-node/register --timeout 500000 'test/mocha/**/*.ts'",
+    "test:ts": "forge build --force && npx mocha --require ts-node/register --timeout 500000 'test/mocha/**/*.ts'",
```

Comment:
The `build-info` file is now required for our tests since we recompile the contracts during the deploy/proposal simulation steps. To speed up the tests, I had them read in the file directly using JSON.parse, which is significantly faster than using the normal `makeGetConfigArtifacts` function. This requires exactly one build info file, though, so I added `--force` here to ensure that's always the case.

Unfortunately, the changes in this PR do result in our tests being slightly slower overall due to the added burden of recompiling all the contracts. With these changes, our plugin test suite now takes about 8.5 minutes vs 6.5 previously. It's not ideal, but it's not a terrible regression all things considered.